orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepository.kt

```diff
@@ -419,8 +423,8 @@ class SqlExecutionRepository(
     return retrieve(type, criteria, null)
   }
 
-  private fun retrieve(type: ExecutionType, criteria: ExecutionCriteria, partition: String?): Observable<PipelineExecution> {
-    withPool(readPoolName) {
+  private fun retrieve(type: ExecutionType, criteria: ExecutionCriteria, partition: String?, includeNestedExecutions: Boolean = false): Observable<PipelineExecution> {
+    withPool(poolName) {
```

Comment:
What's the rationale for this change? I would have thought we would want to be using the read pool here?

Reply:
Yeah I SUSPECT this was a confusion on this.  There's a risk possibly on chained executions on concurrency but... still think should be readPoolName on this one...

Reply:
This is a mistake I did in the code, the right pool should be readPoolName.